packages/core/src/index.ts

```diff
@@ -294,37 +315,33 @@ export class IDX {
     return index?.[key] ?? null
   }
 
-  async _setReference(key: IndexKey, id: DocID): Promise<void> {
-    await this._indexProxy.changeContent<Index>((index) => {
+  async _setReference(key: IndexKey, id: StreamID): Promise<void> {
+    await this._indexProxy.changeContent((index) => {
       return { ...index, [key]: id.toUrl() }
     })
   }
 
   async _setReferences(references: Index): Promise<void> {
     if (Object.keys(references).length !== 0) {
-      await this._indexProxy.changeContent<Index>((index) => {
+      await this._indexProxy.changeContent((index) => {
         return { ...index, ...references }
       })
     }
   }
 
   async _removeReference(key: IndexKey): Promise<void> {
-    await this._indexProxy.changeContent<Index>(({ [key]: _remove, ...index }) => {
+    //await this._indexProxy.changeContent(({ [key]: _remove, ...index }) => {
```

Comment:
Any reason for the change here please?

Reply:
Yeah types were not happy about the way it was before :/